frontend/src/js/standard-query-editor/QueryNode.tsx

```diff
@@ -180,37 +187,39 @@ const QueryNode = ({
       : undefined;
 
   const QueryNodeRoot = (
-    <Root
-      ref={(instance) => {
-        ref.current = instance;
-        drag(instance);
-      }}
-      active={hasNonDefaultSettings || hasFilterValues}
-      onClick={node.error ? undefined : () => onEditClick(andIdx, orIdx)}
-    >
-      <QueryNodeContent
-        error={node.error}
-        isConceptQueryNode={nodeIsConceptQueryNode(node)}
-        tooltipText={tooltipText}
-        label={label}
-        description={description}
-        rootNodeLabel={rootNodeLabel}
-      />
-      <QueryNodeActions
-        andIdx={andIdx}
-        orIdx={orIdx}
-        excludeTimestamps={node.excludeTimestamps}
-        isExpandable={isQueryExpandable(node)}
-        hasActiveSecondaryId={hasActiveSecondaryId}
-        excludeFromSecondaryId={node.excludeFromSecondaryId}
-        onDeleteNode={onDeleteNode}
-        onToggleTimestamps={onToggleTimestamps}
-        onToggleSecondaryIdExclude={onToggleSecondaryIdExclude}
-        onExpandClick={expandClick}
-        previousQueryLoading={node.loading}
-        error={node.error}
-      />
-    </Root>
+    <FlexHoverNavigatable triggerNavigate={onClick}>
```

Comment:
If we're adding this to `QueryNode`, let's also add it to the concepts in the forms, so to `FormConceptNode`